BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/LocationView.swift

```diff
@@ -399,7 +399,6 @@ final class LocationView: UIView, LocationTextFieldDelegate, ThemeApplicable, Ac
         lockIconButton.accessibilityLabel = state.lockIconButtonA11yLabel
 
         urlTextField.accessibilityIdentifier = state.urlTextFieldA11yId
-        urlTextField.accessibilityLabel = state.urlTextFieldA11yLabel
```

Comment:
Removing the a11y label means that when a website is entered in the `LocationView`, it will read out just the website name, and not the website name AND "search or enter address"